website/static/js/components/legend.js

```diff
@@ -1,17 +1,56 @@
 require('css/legend.css');
 var m = require('mithril');
 
+/** Finds the index of a string in an array
```

Comment:
Find index probably is, since I think JS provides .indexOf, however remove_index was the only way I could figure out a static way of removing an index without leaving a misindexed arrays or missing values.